papers/libceed-paper/paper.rst

```diff
@@ -0,0 +1,72 @@
+:author: Valeria Barra
+:email: valeria.barra@colorado.edu
+:institution: University of Colorado Boulder
+
+:author: Jed Brown
+:email: jed@jedbrown.org
+:institution: University of Colorado Boulder
+
+:author: Jeremy Thompson
+:email: jeremy.thompson@colorado.edu
+:institution: University of Colorado Boulder
+
+:author: Yohann Dudouit
+:email: dudouit1@llnl.gov
+:institution: Lawrence Livermore National Laboratory
+
+:bibliography: bibliography
+
+----------------------------------------------------------------------------------
+High-performance operator evaluations with ease of use: libCEED's Python interface
+----------------------------------------------------------------------------------
+
+.. class:: abstract
+
+   libCEED is a new open-source, lightweight library designed to leverage the next generation exascale machines by allowing a wide variety of applications to share highly optimized kernels. libCEED offers implementations, selectable at runtime, tuned for a variety of computational device types, including CPUs and GPUs. libCEED’s purely algebraic framework can unobtrusively be integrated in new and legacy software to provide performance portable applications. In this work, we present libCEED's newly available Python interface that opens up new strategies for parallelism and scaling in high-performance Python, without having to compromise ease of use.
+
+.. class:: keywords
+
+   High-performance Python, performance portability, scalability, parallelism
```

Comment:
Don't we want "Finite Element" and "High order" in the keywords?

Reply:
Thanks. I retrieved the ones used on easychair for the abstract submission. I don't recall now if there was a list to choose from. I think we could add some more, and hopefully the reviewers won't complain if they do not exactly match the abstract submission ones.